core/service/src/lib.rs

```diff
@@ -457,9 +425,9 @@ impl<Components: components::Components> Service<Components> {
 						"implementation" => impl_name.clone(),
 						"version" => version.clone(),
 						"config" => "",
+						"aura" => aura,
```

Comment:
That's the data that is sent to the telemetry server. Modifying it means that the backend needs to be updated as well. Additionally, I'd really prefer if what was in the `aura` and `grandpa_voter` variables had a well-defined structure, so as to not break the telemetry.